buildconfig/Jenkins/check_for_changes

```diff
@@ -36,6 +36,16 @@ case "$TYPE" in
         fi
         exit $NOTFOUND
     ;;
+    docs-gui-only)
+	# FOUND=1 iff changes are limited to docs or GUI only
+	# Find all changed files and grep for required type. -v inverts match so grep=0 means
+	# there are other changes besides this
+	if git diff --name-only ${BRANCH_TIP} ${BRANCH_BASE} -- | grep -q -E -v '^docs/|^MantidQt/|^MantidPlot/'; then
+	    exit $NOTFOUND
+	else
+	    exit $FOUND
```

Comment:
Thanks for catching that. I hadn't checked the system test builds yet.

Yes, you're right. We should `exit 1` if we have something other than those things changed so that then the outer `buildscript` will take the else branch. :+1: